.github/workflows/bitcoin-tests.yml

```diff
@@ -2,8 +2,10 @@ name: stacks-bitcoin-integration-tests
 
 # Only run when:
 #   - PRs are opened
+#   - the workflow is started from the UI
 on:
   pull_request:
+  workflow_dispatch:
```

Comment:
Nice!
https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch